packages/eslint-plugin/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@reatom/eslint-plugin",
+  "version": "3.1.0",
+  "private": false,
+  "sideEffects": false,
+  "description": "Reatom for eslint-plugin",
+  "source": "src/index.ts",
+  "exports": {
+    "require": "./build/index.js",
+    "default": "./build/index.module.js"
+  },
+  "main": "build/index.js",
+  "module": "build/index.module.js",
+  "unpkg": "build/index.umd.js",
+  "types": "build/index.d.ts",
+  "browserslist": [
+    "last 4 chrome versions"
+  ],
+  "scripts": {
+    "sandbox": "vite",
+    "prepublishOnly": "npm run build && npm run test",
+    "build": "microbundle",
+    "test": "tsx src/**/*.test.ts",
+    "test:watch": "tsx watch src/**/*.test.ts"
+  },
+  "dependencies": {
+    "@reatom/core": ">=3.1.0"
+  },
+  "author": "artalar",
+  "maintainers": [
+    {
+      "name": "artalar",
+      "url": "https://github.com/artalar"
+    }
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/artalar/reatom.git"
+  },
+  "bugs": {
+    "url": "https://github.com/artalar/reatom/issues"
+  },
+  "homepage": "https://github.com/artalar/reatom/packages/eslint-plugin",
+  "files": [
+    "/build",
+    "/package.json"
+  ],
+  "devDependencies": {
```

Comment:
Maybe we should use less strict version, from `8.0.0`? To be much compatible with current stack of a user.

Reply:
It's dev, not peer, but yeah, mb it's better to not use new features)

Reply:
Got it, my mistake)